integration/e2e/step-definitions/content-search.steps.ts

```diff
@@ -25,4 +47,29 @@ defineSupportCode(function ({ Given, When, Then }) {
     expect(firstHit).to.contains(`${term} </em>`);
   });
 
+  Then(/^the content of hit number (.*) should be displayed$/, async (hit) => {
```

Comment:
The title say that the **content** on a given page should be displayed, but there is no check that the content actually is displayed. It only checks that the page number is correct.